### ISSUE TITLE
Update ISSUE and PR template to reflect imperative titles.

### ISSUE THREAD

**Issue by Monal5031** [anshumanv/evolutions-webapp#23]
##### This issue is about {Issue Title}

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

Commits should be in imperative manner, also the PR template shows to use Fixes which is technically only used for bug fixes rather for enhancements, where we use Closes.

**Comment by anshumanv** [comment]
>PR template shows to use Fixes which is technically only used for bug fixes rather for enhancements, where we use Closes.

Keen! :scream:

**Comment by anshumanv** [comment]
@Monal5031 Update.

**Comment by anshumanv** [comment]
Ping @Monal5031
